src/librustc_mir/build/scope.rs

```diff
@@ -107,6 +107,9 @@ pub struct Scope<'tcx> {
     /// `ScopeAuxiliary`, but kept here for convenience
     extent: CodeExtent,
 
+    /// Whether there's anything for the cleanup path.
```

Comment:
We may also not do it because LLVM does weird things with memory dependency along the unwind path. We had to remove some lifetime annotations in a similar situation from old trans, @dotdash should know the details.